core/src/main/java/com/tweet/core/model/User.java

```diff
@@ -13,6 +12,10 @@
     private String firstName;
     private String surname;
 
+    @Enumerated(EnumType.STRING)
+    @ManyToMany
+    private List<UserRole> userRoles;
```

Comment:
Do want user to have multiple same roles? Maybe Set would be more suitable for this :)